build.gradle

```diff
@@ -151,7 +151,3 @@ compileTestKotlin {
         jvmTarget = "11"
     }
 }
-
```

Comment:
If you migrate to AutoService than please add

apply plugin: 'kotlin-kapt'

dependencies {
    annotationProcessor 'com.google.auto.service:auto-service:1.0.1'
    kapt 'com.google.auto.service:auto-service:1.0.1'
}